src/test/java/org/osc/controller/nsc/OSGiIntegrationTest.java

```diff
@@ -42,8 +43,15 @@
 import org.osgi.framework.ServiceObjects;
 import org.osgi.util.tracker.ServiceTracker;
 
-@RunWith(PaxExam.class)
-@ExamReactorStrategy(PerMethod.class)
+/*TODO: test is commented, because there is a problem with openstack4j dependecies injected directly into osgi (whicch is build during that test)
```

Comment:
This is rather missed configuration as we are not good at OSGI - needed few days to investigate missing dependencies to enable test but we didn't have time to finish it till our transfer to new projects. This topic was passed to paremus